lib/components/narrative/connected-trip-details.js

```diff
@@ -20,10 +20,6 @@ const mapStateToProps = (state) => {
       mediumId: null,
       riderCategoryId: null
     },
-    displayCalories:
-      typeof itinerary?.displayCalories === 'boolean'
```

Comment:
Why was this removed?

Reply:
Displaying calories was completely removed from OTP-UI, so this doesn't apply anymore.